Communitter/src/main/java/com/communitter/api/service/PostService.java

```diff
@@ -98,4 +132,8 @@ private boolean checkRequiredFields(Set<PostField> postFields, Template postTemp
         }
         return true;
     }
+
+    public List<Post> getAllPosts() {
```

Comment:
why do we need all post without community or user filter?

Reply:
@ynsmrkrkmz normally we see the posts under community, its for sendind a request from postman to see the informaton regarding post while working with posts.